lib/modules/sections.mjs

```diff
@@ -525,13 +525,15 @@ export async function validateReferencesSection (doc, { mode = MODES.NORMAL } =
       const normativePattern = /normative/i
       const informativePattern = /informative/i
       const subsections = []
-      const plainReferenceFormat = /^\d+\.\s+References$/i
+      const plainReferenceFormat = /^(?:\d+\.\s+)?References$/i
+      const referenceSectionTitlePattern = /^(?:\d+\.\s+)?(?:(?:Normative|Informative)\s+)?References$/i
+      let numberOfSectionTitle = 0
```

Comment:
suggestion to make this easier for future maintainers to follow along: `numberOfReferenceSectionTitles` perhaps?